google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1/doc/google/devtools/clouderrorreporting/v1beta1/common.rb

```diff
@@ -15,6 +15,34 @@
 module Google
   module Devtools
     module Clouderrorreporting
+      ##
+      # The `Google::Devtools::Clouderrorreporting::V1beta1` module provides the following types:
+      #
+      # Class | Description
+      # ----- | -----------
+      # {Google::Devtools::Clouderrorreporting::V1beta1::DeleteEventsRequest} | Deletes all events in the project.
```

Comment:
@dazuma. These descriptions are copy/paste from [the GAPIC source](https://github.com/GoogleCloudPlatform/google-cloud-ruby/blob/master/google-cloud-error_reporting/lib/google/cloud/error_reporting/v1beta1/doc/google/devtools/clouderrorreporting/v1beta1/error_stats_service.rb#L238). (It's manual right now, but Song opened #1180 for generating the table.)